src/main/server/update.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# -----------------------------------------------------------------------------
+# update.sh - Update Script
+# -----------------------------------------------------------------------------
+
+# Description:
+#   This script facilitates the seamless update of Ant Media Server to the latest version,
+#   ensuring you have the most up-to-date features, improvements, and bug fixes. 
+#   For more information: https://github.com/ant-media/Ant-Media-Server/releases
+
+# Usage:
+#   1. Run the script directly in the terminal using "./update.sh".
+#   2. Utilize the dashboard's integrated update feature for a user-friendly experience.
+
+# Note:
+#   - Ensure internet connectivity for downloading the latest Ant Media Server release.
+
+# -----------------------------------------------------------------------------
+
+INSTALL_DIRECTORY=/usr/local/antmedia
+
+REMOTE_VERSION=$(curl -s "https://antmedia.io/rest/VERSION")
+LOCAL_VERSION=$(unzip -p $INSTALL_DIRECTORY/ant-media-server/ant-media-server.jar | grep -a "Implementation-Version"|cut -d' ' -f2 | tr -d '\r')
```

Comment:
I think the path in ant-media-server/ant-media-server.jar not correct.

Reply:
Yes, It 's my bad. Sorry for that. I m still making some changes.